percpu_macros/src/lib.rs

```diff
@@ -3,6 +3,53 @@
 //! **DO NOT** use this crate directly. Use the [percpu] crate instead.
 //!
 //! [percpu]: https://docs.rs/percpu
+//! 
+//! ## Implementation details of the `def_percpu` macro
```

Comment:
Put it in README?

Reply:
Is is OK to implementation details to README?